simple/entities/README.md

```diff
@@ -0,0 +1,25 @@
+# Simple Entities Importer
```

Comment:
Also, why is it called entities importer?  This is Simple Observations Importer really?

Reply:
Eventually, we might have more importers for different types of data and they will all be producing observations. This one is called entities importer since it resolves entities using the resolution APIs. I'm open to other names though.

Reply:
Hmmm, the reason I wonder is, this is importing "observations", whereas, importing entities (like triples in the graph like food banks) is what we've been calling "entities" import.

Reply:
Renamed to `SimpleStatsImporter` after an offline chat.